lib/hanami/assets/helpers.rb

```diff
@@ -186,9 +240,25 @@ def javascript(*sources)
       #   <%= stylesheet 'application' %>
       #
       #   # <link href="https://assets.bookshelf.org/assets/application-28a6b886de2372ee3922fcaf3f78f2d8.css" type="text/css" rel="stylesheet">
-      def stylesheet(*sources)
+      def stylesheet(*sources, **options)
         _safe_tags(*sources) do |source|
-          html.link(href: _typed_asset_path(source, STYLESHEET_EXT), type: STYLESHEET_MIME_TYPE, rel: STYLESHEET_REL).to_s
+          tag_options = options.merge({
+            href: _typed_asset_path(source, STYLESHEET_EXT),
+            type: STYLESHEET_MIME_TYPE,
+            rel: STYLESHEET_REL
+          })
+
+          if _sri? || tag_options.include?(:integrity)
+            tag_options[:integrity] ||= _sri_value(source, STYLESHEET_EXT)
+            tag_options[:crossorigin] ||= CROSSORIGIN_ANONYMOUS
+          end
+
+          html.link(
+            href: tag_options.delete(:href),
```

Comment:
@jodosha if I am going to be changing the tests, I have a little more scope with the implementation. There are a couple of things that I would like to change, for example, `Hanami::Assets::Bundler#checksum` actually copies the asset file to the digest, which isn't implied by the method. I think that I can refactor `#run` with a couple of more steps which will allow a cleaner implementation of the digest and SRI features. Are you happy for me to go ahead an do this?